src/locales/ar.po

```diff
@@ -720,6 +733,11 @@ msgstr "رجاء أدخل عنوان بريدك الإلكتروني."
 msgid "Please log in."
 msgstr "الرجاء تسجيل الدخول."
 
+#, fuzzy
+#| msgid "Please note that your settings have been updated as follows:"
+msgid "Please note that client %1 settings has been updated as below:"
+msgstr "يرجى ملاحظة أن الإعدادات الخاصة بك تم تحديثها كما يلي:"
```

Comment:
yes it will be marked as need review in weblate and string marked as fuzzy will not be translated as maketext ignores them, they will be displayed as english text as in msgid